packages/ia-components/sandbox/track-lists/track-utils.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * For a given track in an album, it formats the track title value
+ * @param { string } trackTitle
+ * @param { string } fileName
+ */
+const formatTrackTitle = (trackTitle = '', fileName = '') => {
+  if (trackTitle) {
+    return trackTitle;
+  }
+  // strip file extension from its name and return
+  return fileName.replace(/.mp3/, '');
```

Comment:
Are we skipping a mutation to sentence structure because we can't guarantee a file name formatting standard? I was thinking with the unit test you wrote for this, you could make it more human readable with
```
fileName.replace(/^(.)/, (search, $1) => $1.toUpperCase()).replace(/(_)(\w)/g, (search, $1, $2) => ` ${$2.toUpperCase()}`).replace(/\.mp3$/, '');
```

Reply:
yes, we currently are skipping mutations to sentence structure at this point 👍